v03_pipeline/lib/tasks/write_tdr_metrics_file.py

```diff
@@ -0,0 +1,35 @@
+import csv
+
+import luigi
+import luigi.util
+
+from v03_pipeline.lib.misc.terra_data_repository import (
+    BIGQUERY_METRICS,
+    bq_metrics_query,
+)
+from v03_pipeline.lib.paths import tdr_metrics_path
+from v03_pipeline.lib.tasks.base.base_loading_pipeline_params import (
+    BaseLoadingPipelineParams,
+)
+from v03_pipeline.lib.tasks.files import GCSorLocalTarget
+
+
+@luigi.util.inherits(BaseLoadingPipelineParams)
+class WriteTDRMetricsFileTask(luigi.Task):
+    bq_table_name = luigi.Parameter()
```

Comment:
The idea is to write the "metrics" files out as tsvs, one per "TDR dataset".  We can then import them into the pipeline's existing sex check structure.  Later on, we can tweak that to be less reliant on hail and more performant.